packages/hadron-build/lib/target.js

```diff
@@ -638,24 +647,27 @@ class Target {
     const rhelArch = this.arch === 'x64' ? 'x86_64' : 'i386';
     const rhelCategories = _.get(platformSettings, 'rpm_categories');
     this.linux_rpm_filename = `${this.slug}-${this.version}.${rhelArch}.rpm`;
-
-    var isRhel = process.env.EVERGREEN_BUILD_VARIANT === 'rhel';
-    this.linux_tar_filename = `${this.slug}-${this.version}-${
-      isRhel ? 'rhel' : this.platform
-    }-${this.arch}.tar.gz`;
+    this.linux_tar_filename = `${this.slug}-${this.version}-${this.platform}-${this.arch}.tar.gz`;
+    this.rhel_tar_filename = `${this.slug}-${this.version}-rhel-${this.arch}.tar.gz`;
```

Comment:
how did this work ever? did it got renamed when we uploaded on the internal S3?

Reply:
It did not for awhile now, but also these files are not shown in download center and probably nobody ever used them, so we never got any error reports about this